layer/contracts/tracker.py

```diff
@@ -149,38 +149,41 @@ def __str__(self) -> str:
 
 @unique
 class AssetTrackerStatus(str, Enum):
+    """
+    PENDING -> BUILDING / TRAINING -> ASSERTING -> ASSERTED -> UPLOADING -> DONE
+    DOWNLADING -> FROM CACHE / LOADED
+    """
```

Comment:
how can we make sure this doesn't go obsolete? is it testable?

Reply:
it's more for internal documentation than anything else. i found that this logic existed in a different part of the codebase so I wanted to surface it here. I can't think of an easy way of testing this.